### DISCUSSION TITLE
What is the License of this repo?

### DISCUSSION THREAD

**Discussion by abdala** [scottyzen/woonuxt#147] [General]
I couldn't find a License file. How can we use this code?

**Comment by scottyzen** [comment]
I haven't added a license file yet, but you can use this code for personal or commercial use. I'll add a license file soon. It's open-source, you can use it for your clients. However, at the very minimum, I would appreciate it if you could keep the credit in the footer.

**Reply by abdala** [reply]
Nice! Thanks!
